src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -55,6 +55,8 @@ const messages = defineMessages({
   plexwatchlistsyncseries: 'Auto-Request Series',
   plexwatchlistsyncseriestip:
     'Automatically request series on your <PlexWatchlistSupportLink>Plex Watchlist</PlexWatchlistSupportLink>',
+  collapsetags: 'Collapse Tags',
+  collapsetagstip: 'Collapse tags by default in Movie/Series detail page',
```

Comment:
Think this sounds slightly better: Collapse tags by default on the Movie/Series details page